### PR TITLE
cxxopts/all adding check before min cppstd

### PR DESCRIPTION
**Pull request by prince-chrismc**
Specify library name and version:  **cxxpots/all**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by rpoisel** [comment]
Sorry for being picky, but the title of this pull request is a bit misleading.

**Comment by prince-chrismc** [comment]
Thanks for noticing! I'm moving and working mobile for the most part. Not fun.

**Review by jgsogo — DISMISSED** [review]

File: recipes/cxxopts/all/conanfile.py [hunk 1]
```diff
@@ -13,10 +14,35 @@ class CxxOptsConan(ConanFile):
...
+
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "14",
+            "gcc": "6",
```
Comment (uilianries) [inline recipes/cxxopts/all/conanfile.py:30]:
```suggestion
            "gcc": "5",
```

GCC 5 is more than enough for C++11

File: recipes/cxxopts/all/conanfile.py [hunk 2]
```diff
@@ -13,10 +14,35 @@ class CxxOptsConan(ConanFile):
...
+            "clang": "4",
```
Comment (uilianries) [inline recipes/cxxopts/all/conanfile.py:31]:
```suggestion
            "clang": "3.9",
```

Clang also is more than enough for C++11

https://en.cppreference.com/w/cpp/compiler_support#cpp11

**Review by uilianries — CHANGES_REQUESTED** [review]

File: recipes/cxxopts/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/cxxopts/all/conanfile.py:30]: https://github.com/conan-io/conan-center-index/pull/2905#issuecomment-692029125 thats what i tried originally :thinking:

Comment (prince-chrismc) [reply recipes/cxxopts/all/conanfile.py]: its was bad cmake in the test_package

**Review by jgsogo — APPROVED** [review]

**Review by intelligide — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**